x-pack/plugins/observability_solution/apm/public/components/alerting/ui_components/popover_expression.tsx

```diff
@@ -14,10 +14,11 @@ interface Props {
   value: React.ReactNode;
   children?: React.ReactNode;
   color?: ExpressionColor;
+  dataTestSubj?: string;
```

Comment:
I wonder if it would be better to name it `data-test-subj` instead?

Reply:
I used `dataTestSubj` since this is our component not Eui, but I can change it if that's preferred. I also saw `dataTestSubj` used in other places in APM plugin such as [this](https://github.com/elastic/kibana/blob/main/x-pack/plugins/observability_solution/apm/public/components/shared/suggestions_select/index.tsx#L24).

Reply:
Does it mean when I want to access this prop, I should do something like:

```
const { title, value, children, color, 'data-test-subj': dataTestSubj } = props;
```

or

```
data-test-subj={props['data-test-subj']}
```

The current implementation seems simpler, IMO.

Reply:
IMO this attribute should preferably be all in either kebab-case or cameCase, but both conventions are used all over Kibana. My comment was a nit though. Anything you decide will be fine with me.